.gitignore

```diff
@@ -18,3 +18,9 @@ node_keys.txt
 /resources/gemini-3c/full_node_keys.txt
 /resources/gemini-3c/rpc_node_keys.txt
 /resources/gemini-3c/relayer_ids.txt
+/resources/devnet/relayer_ids.txt
```

Comment:
What is relayer doing here?

Reply:
this hold the relayer ids used to run the relayers for domains